Kconfig.macros

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
```

Comment:
I think it's better if we create a folder and move this file to `/kconfig/Kconfig.consts`

Reply:
Done.